easybuild/easyconfigs/p/PLUMED/PLUMED-2.2.2-intel-2016a.eb

```diff
@@ -0,0 +1,42 @@
+# by Ward Poelmans <wpoely86@gmail.com>
+
+easyblock = 'ConfigureMake'
+
+name = 'PLUMED'
+version = '2.2.2'
+
+homepage = 'http://www.plumed-code.org'
+description = """PLUMED is an open source library for free energy calculations in molecular systems which
+ works together with some of the most popular molecular dynamics engines. Free energy calculations can be 
+ performed as a function of many order parameters with a particular  focus on biological problems, using
+ state of the art methods such as metadynamics, umbrella sampling and Jarzynski-equation based steered MD.
+ The software, written in C++, can be easily interfaced with both fortran and C/C++ codes.
+"""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+toolchainopts = {'usempi': 'True'}
+
+source_urls = ['https://github.com/plumed/plumed2/archive/']
+sources = ['v%(version)s.tar.gz']
+
+dependencies = [
+    ('zlib', '1.2.8'),
+    ('GSL', '2.1'),
+    ('libmatheval', '1.1.11'),
+]
+
+preconfigopts = 'FC=$MPIF90 LIBS="$LIBLAPACK $LIBS" '
```

Comment:
other than this remark, this PR should be good to go, please ping me when it's tackled